github-repos/src/components/RepoList/RepoList.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { connect } from "react-redux";
+
+const ReposList = ({ repos }) => {
+  const list = repos.map((repo, index) => {
```

Comment:
Repos already have an ID that comes with them. Do you think that using the repo ID as the key value in `<li key={}>` would work instead of using normal indices?